### DISCUSSION TITLE
400 Bad Request when download

### DISCUSSION THREAD

**Discussion by mashirozx** [chevereto/docker#52] [Q&A]
<img width="654" alt="image" src="https://user-images.githubusercontent.com/16148054/219535864-62120088-4e21-47c7-8cdd-f93ce6e0d86f.png">

**Comment by mashirozx** [comment]
Both with/without license input returns this.

IDC: Oracle Integration Cloud (OIC)
CIDR: 138.2.0.0/16
NetRange: 138.2.0.0 - 138.2.255.255

**Comment by rodber** [comment]
When accessing to the API download and not providing the key, the system redirects (http redirect) to GitHub download link.

HTTP 4XX namespace is for client error, you should check how your machine is connecting to internet.

**Comment by Griefed** [comment]
Only calling the script won't do, @mashirozx 

You must call multiple commands, as stated in to docs: https://v4-docs.chevereto.com/guides/docker/pure-docker.html#build-image

The `400 Bad Request` is occuring because you are not supplying a version to the script.
